src/utils/pick.js

```diff
@@ -2,7 +2,7 @@
 
 export default function pick(object: Object, keys: Array<string>): Object {
   return keys.reduce((obj, key) => {
-    if (object[key]) {
+    if (object[key] !== undefined) {
```

Comment:
Could you replace this by `typeof object[key] !== 'undefined'` please?

Reply:
Sure thing, good suggestion! https://github.com/PaulLeCam/react-leaflet/pull/649/commits/88db6a509bbd6b2f91ef873a1fd87338d4185aa0